### PR TITLE
DVC-8772 chore: filter prompts for overrides

### PR DESCRIPTION
**Pull request by suthar26**
- tested locally by changing the number of overrides through dashboard 

Screenshot 2023-09-11 at 2 40 44 PM

### CODE REVIEW

**Comment by suthar26** [comment]
Current dependencies on/for this PR:
* main
  * **PR #284** <a href="https://app.graphite.dev/github/pr/DevCycleHQ/cli/284" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/DevCycleHQ/cli/284?utm_source=stack-comment).

File: src/ui/prompts/overridePrompts.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+            }
+            if (choices.find((choice: FeatureChoice) => choice.value._id === feature._id)) {
+                continue
+            }
+            choices.push({
+                name: feature.name || feature.key,
+                value: feature
+            })
+        }
+        return autocompleteSearch(choices, search)
```
Comment (chantalchung) [inline src/ui/prompts/overridePrompts.ts:33]: I think we need some kind of empty state message if there are no features with overrides (`choices` array is empty) - same case for environments below
image
I'm stuck on this step when I have no overrides

Comment (suthar26) [reply src/ui/prompts/overridePrompts.ts:33]: yess will add!

Comment (suthar26) [reply src/ui/prompts/overridePrompts.ts:33]: this is the same thing that happens when you create a project with no features and do `feature update` the list is empty.. couldn't find a way to make it not not execute here without making an extra call to the api

Comment (kaushalkapasi) [reply src/ui/prompts/overridePrompts.ts:33]: just escape early, if no results found then return and say `No Overrides found.` with an info print

Comment (suthar26) [reply src/ui/prompts/overridePrompts.ts:33]: added empty states

**Pull request merged**